### PR TITLE
SDK-440 [Java SDK] Release v4.2.0 - Update of readme.md

### PR DESCRIPTION
**Pull request by vadim-rutkevich-incountry**
Work relates to SDK-440 - Update of readme.md

### CODE REVIEW

**Review by VladimirLevitskiy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -770,25 +772,25 @@ public interface Storage {
...
  ```java
 String recordKey = "user_1";
 storage.delete("us", recordKey);
  ```
 
-### Batch delete records
+### Deleting multiple records
 
-Use `batchDelete` method in order to delete multiple records from the InCountry storage. For now, SDK allows deletion only by a list of `RecordKey` values (provided via `FindFilter` instance).
+You can use the `batchDelete` method in order to delete multiple records from the InCountry platform. As of now, the SDK supports the deletion of records only by providong a list of the `RecordKey` values passed via the `FindFilter` instance.
```
Comment (RommeroInCountry) [inline README.md:783]: providong - typo

**Review by RommeroInCountry — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1163,7 +1165,7 @@ public class CustomEncryptionKey extends Secret {
 }
 ```
 
-You can set the `isForCustomEncryption` parameter by using the `SecretsData` method in the JSON format, as follows:
+You can set the `isForCustomEncryption` parameter in the JSON format within the `SecretsData` method , as follows:
```
Comment (RommeroInCountry) [inline README.md:1168]: You can also specify a key for custom encryption by setting the `isForCustomEncryption` parameter in `SecretsData` JSON representation:

**Review by RommeroInCountry — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RommeroInCountry) [reply README.md:783]: as of now -> for now

File: README.md (hunk 2 above)
Comment (vadim-rutkevich-incountry) [reply README.md:1168]: updated

**Review by RommeroInCountry — APPROVED** [review]

**Review by VladimirLevitskiy — APPROVED** [review]

**Pull request merged**